pages/publishing/publishing-use-cases.mdx

```diff
@@ -0,0 +1,42 @@
+---
+layout: 'guide'
+title: 'Publishing Use Cases'
+# abstract: 'The use cases for publishing your tracking plan from Avo to your analytics tool or CDP'
+---
+
+import Link from '../../components/Link';
+
+<p>
+  <img
+    src={require('../../images/publishing.svg')}
+    width="200px"
+    alt="Avo Publishing"
+/>
+
+</p>
+
+## See event and property descriptions from Avo inline in your analytics tool
+
+Having the description at hand when navigating your events in your analytics platform, like Mixpanel or Amplitude, helps you more easily identify the events and properties you’re looking for when building charts and looking at dashboards.
+With an Avo publishing integration you can let Avo automatically push description and other tracking plan details into these tools, so you can have more context at hand when using the data.
+
+## Keep data validation in your CDP up to date with the tracking plan
+
+Segment Protocols, mParticle Data Master, RudderStack Tracking Plans and Snowplow Iglu are all examples of CDPs with ingestion time validation. It allows you to validate the events as they come in, and for example route the bad events to alternative destinations.
+With Avo publishing you can make sure the schemas used for this validation are always up to date with your tracking plan in Avo, which increases the quality of your data in all downstream destinations.
+Being able to integrate with CDPs this way also allows your teams to gradually adopt Avo over time.
+
+## Receive a webhook every time the Avo tracking plan is updated
+
+The publishing webhook is a building block that allows you to integrate your Avo tracking plan with your other schema registries and internal systems, tools and workflows.
+What can you build with the webhook?
+Here are some ideas to get you started:
+
+- ### **Update ingestion time validation:**
+  _Automatically update your JSON Schema ingestion time validation whenever a tracking plan change is merged to the main branch. Making sure your pipeline is ready to approve the fresh events before they arrive._
+- ### **Manage tables in data warehouses:**
+  _Automatically keep raw event table schemas in SnowflakeDB, BigQuery or Redshift in sync with the Tracking Plan – keeping your data streaming in smoothly_
+- ### **Generate dbt models:**
+  _Generate .yml config files describing your raw event tables in dbt. I'm also excited to explore the possibility of generating detailed dbt tests based on the Tracking Plan spec_
```

Comment:
> I'm also excited to explore […]

👂🏼 Is that the Avo person speaking to me? 🤠 It feels a bit strange having this part personal. Should we make it generic like the rest of the text?

Reply:
Lol i did not notice that. Guilty not reading the text very much – was focusing on restructure 😅